spec/javascripts/health-care/components/personal-information/VeteranAddressSection.spec.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+
+import VeteranAddressSection from '../../../../../_health-care/_js/components/personal-information/VeteranAddressSection';
+
+describe('<VeteranAddressSection>', () => {
+  const nullAddress = [{
+    street: null,
+    city: null,
+    country: null,
+    state: null,
+    zipcode: null,
+  }];
+
+  it('Sanity check the component renders', () => {
+    const tree = SkinDeep.shallowRender(<VeteranAddressSection data={{ address: nullAddress }}/>);
+    const vdom = tree.getRenderOutput();
+    expect(vdom).to.have.property('type', 'fieldset');
```

Comment:
So I am not super well versed in all of this but it is my understanding that `getRenderOutput()` will render whatever your element blob is and to check the property `type` you will always return whatever the wrapper is for the component. If you want to check for other things then we will need to check for children inside of the blob. I also am seeing in this case that `fieldset ` was a wrapper on top of a `div` wrapper. I don't know if that was a styling decision or not but it seems un necessary and in that case each section would be wrapped in a `div`.